applications/FluidDynamicsApplication/custom_elements/qs_vms_dem_coupled.h

```diff
@@ -249,10 +249,26 @@ class QSVMSDEMCoupled : public QSVMS<TElementData>
 
     // Protected interface of FluidElement ////////////////////////////////////
 
+    void AlgebraicMomentumResidual(
+        const TElementData& rData,
+        const array_1d<double,3> &rConvectionVelocity,
+        array_1d<double,3>& rResidual);
+
+    void MomentumProjTerm(
+        const TElementData& rData,
+        const array_1d<double,3>& rConvectionVelocity,
+        array_1d<double,3> &rMomentumRHS);
```

Comment:
I think that you need to add the `override` specifier in here (I got a warning when compiling in my machine).